package-portable-linux.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+## targeting old glibc
+# docker run --mount type=bind,source="$(pwd)",target="/usr/src/app" -it ubuntu:16.04
```

Comment:
This isn't fully automated yet, but the `objdump -T <binfile> | grep GLIBC_ | awk '{print $(NF-1)}' | sort -V` output suggests cimbar/cimbar_send built this way will work on glibc 2.14(!) and up. 16.04 is 2.23, so that's probably the more trustworthy number -- but who knows.

That said, I did test on centos7 a bit, and it works. So that's cool.